seabird.proto

```diff
@@ -166,6 +167,14 @@ message StreamInfoResponse {
 
   map<string, CommandMetadata> commands = 3;
 
-  // TODO: find a way to track IPs related to plugins - maybe deny connections
-  // from an IP other than the registration
+  string ip = 4;
+  uint64 connection_time = 5;
+}
+
+message CoreInfoRequest {
```

Comment:
Nit: for empty messages, have the `}` on the same line as the `{`

Reply:
Will do. Is there such a thing as `protofmt`?